### ISSUE TITLE
Qt 4 still used as default

### ISSUE THREAD

**Issue by pmattern** [lxde/obconf-qt#8]
ObConf-Qt code is still considering Qt 4 the default version to be used.

Building ObConf-Qt on Arch Linux i686 or x86_64 using latest Git code of all LXQt components yields the following results:
=> Qt 4 available
```
$ cmake <path to source code>
[...]
-- Found Qt4: /usr/bin/qmake-qt4 (found version "4.8.6") 
-- Building with Qt4.8.6
[...]
```
and ObConf-Qt gets built against Qt 4.
=> Qt 4 missing, Qt 5 available
```
$ cmake <path to source code>
[...]
CMake Error at /usr/share/cmake-3.2/Modules/FindQt4.cmake:1326 (message):
  Found unsuitable Qt version "5.4.1" from /usr/bin/qmake, this code requires
  Qt 4.x
```
and cmake exits. After adding -DUSE_QT5=ON ObConf-Qt gets built against Qt 5.
Both times the resulting binary is working as usual.

On Arch Linux qmake is a symbolic link pointing to qtchooser
```
# ls -Al /usr/bin/qmake* 
lrwxrwxrwx 1 root root  9 13. Dez 18:50 /usr/bin/qmake -> qtchooser
lrwxrwxrwx 1 root root 22 14. Dez 18:45 /usr/bin/qmake-qt4 -> /usr/lib/qt4/bin/qmake
lrwxrwxrwx 1 root root 21 26. Feb 19:20 /usr/bin/qmake-qt5 -> /usr/lib/qt/bin/qmake
```
which was selecting Qt 5 all the time when cmake commands as stated above were invoked.
IMHO all this doesn't matter, though, as cmake does seem to detect both symbolic links qmake and qmake-qt4 but chooses the wrong one.

I don't know what the intended behaviour regarding Qt versions looks like, and if e. g. Qt 4 support should actually have been dropped as with all other LXQt components.
But a message like `this code requires Qt 4.x` is definitely wrong and given that Qt5 is the only supported version by LXQt itself I think it shold be chosen by ObConf-Qt as default as well.

**Comment by agaida** [comment]
I think we should default to Qt 5 - and let the Qt4 code working - that would be the most sensible approach if some people want to use compton-conf in environments other than LXQt

**Comment by pmattern** [comment]
Fixed in lxde/obconf-qt@f046ce6.
